src/pages/Blog/Blog.tsx

```diff
@@ -14,17 +20,37 @@ const Blog: React.FC = () => {
         {/*questionable if the search will be performed within the component pagination */}
       </div>
       <div>Pagination component</div>
-      <div>
-        <div>
-          <h2>Suscríbete</h2>
-          <p>deseas estar mas pendiente de más articulos</p>
-          <div>
-            <input type="email" />
-            <button>Suscríbete</button>
+      <Section type="margin" size="sm">
```

Comment:
move all this section to another component, to keeo clean blog main page

Reply:
done